_docs/manage/node.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: guide
+title: StorageOS Docs - Node diagnostics
+anchor: manage
+module: manage/node
+---
+
+# Node
+
+Node shows a per node breakdown of your storage cluster.
+
+## Overview
+
+## Listing node information
+
+To view the state of your cluster simply run:
+```bash
+storageos node ls
+
+NAME                  ADDRESS             HEALTH              SCHEDULER           VOLUMES             TOTAL               USED                VERSION             LABELS
+vol-test-2gb-lon101   46.101.50.155       Healthy 2 days      true                M: 0, R: 2          77.43GiB            5.66%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon102   46.101.50.231       Healthy 2 days      false               M: 1, R: 0          38.71GiB            5.90%               0.7 (00ab7b3 rev)
+vol-test-2gb-lon103   46.101.51.16        Healthy 2 days      false               M: 1, R: 1          77.43GiB            5.61%               0.7 (00ab7b3 rev)
+```
+
+From the output of the command we observe that we have a 3 node StorageOS cluster and the following information:
+- IP addresses of the controller nodes
+- The health of the cluster and uptime
+- Whether the node is a scheduler node or not
+- Number of master copies of volumes or replicas
+- Total capacity in node
+- Percentage of capacity that is in use
+- The version of StorageOS installed
+
+
+## Inspect a node:
+
+For a more detailed overview of a node's status you can inspect it, 
+for example, let's inspect node 3
```

Comment:
Use declarative sentences. "To view detailed information such as state, port configuration, pool membership, health, version and capacity in JSON format, inspect the node:" and remove the line below.